red-pill-contracts-py/src/findings.py

```diff
@@ -0,0 +1,148 @@
+"""Format the agent findings into Forta alerts"""
```

Comment:
this file contains a bunch of items unrelated to redpill. can those be removed?

Reply:
yup